css-conditional-5/Overview.bs

```diff
@@ -64,6 +94,11 @@ Introduction</h2>
 	such that only the first matching rule in a [=conditional rule chain=]
 	is applied.
 
+	It also adds Container Queries.
```

Comment:
I would suggest changing the order of the description of the three new pieces (font technology queries, `@when`/`@else`, and Container Queries) of the introduction.  Specifically:

- I think the bit about querying for supported font technologies should be last, since it's probably the smallest both in scope and size.

- I think the addition of container queries could *optionally* be before the introduction of `@when` and `@else` since I think CQ is a more significant feature but `@when`/`@else` could perhaps be described as having broader scope over the features of the module, so I could see describing these in either order.